src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,430 @@
+| ChunkTransformationMatrix     | RECOMMENDED if `<chunk-index>` is used in filenames | [array][] of [arrays][] of [numbers][] | 3x3 or 4x4 affine transformation matrix describing spatial chunk transformation, for 2D and 3D respectively (for examples: `[[2, 0, 0], [0, 3, 0], [0, 0, 1]]` in 2D for 2x and 3x scaling along the first and second axis respectively or `[[1, 0, 0, 0], [0, 2, 0, 0], [0, 0, 3, 0], [0, 0, 0, 1]]` in 3D for 2x and 3x scaling along the second and third axis respectively). Note that non-spatial dimensions like time and channel are not included in the transformation matrix. |
+| ChunkTransformationMatrixAxis | REQUIRED if `ChunkTransformationMatrix` is present  | [arrays][] of [strings][]              | Describe the axis of the ChunkTransformationMatrix (for examples: `["X", "Y"]` or `["Z", "Y", "X"]`)                                                                                                                                                                                                                                                                                                                                                                                   |
+
+An example of chunk transformations JSON metadata for `chunk-01` and `chunk-05` of Figure 2
+is shown below:
+
+![Figure 2](images/microscopy_chunks2.png "Example figure for chunks transformations")
+
+Figure 2: Example figure for chunks transformations.
+
+In this example, there is no scaling and `chunk-01` is at the origin.
+`chunk-05` is translated by 5 um along `X+` and by 3 um along `Y+`.
+
+**`*_chunk-01_<modality_suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 0],
+                                      [0, 1, 0],
+                                      [0, 0, 1]],
+        "ChunkTransformationMatrixAxis": ["X", "Y"]
+}
+```
+
+**`*_chunk-05_<modality_suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 5],
+                                      [0, 1, 3],
+                                      [0, 0, 1]],
+        "ChunkTransformationMatrixAxis": ["X", "Y"]
+}
+```
+
+#### Example of sidecar JSON file (`*_<modality_suffix>.json`)
+```JSON
+{
+        "Manufacturer": "Hamamatsu",
+        "ManufacturersModelName": "C9600-12",
+        "PixelSize": [0.23, 0.23],
+        "PixelSizeUnits": "um",
+        "Magnification": 40,
+        "BodyPart": "BRAIN",
+        "BodyPartDetails": "corpus callosum",
+        "SampleEnvironment": "exvivo",
+        "SampleFixation": "4% paraformaldehyde, 2% glutaraldehyde",
+        "SampleStaining": "LFB",
+        "SliceThickness": 5,
+        "TissueDeformationScaling": 97
+}
+```
+
+## Required Samples file (`samples.tsv`)
+
+For Microscopy data, the [Samples file](../03-modality-agnostic-files.md#samples-file) is REQUIRED.
```

Comment:
```suggestion
For Microscopy data, the [Samples file](../03-modality-agnostic-files.md#samples-file) `sample.tsv` is REQUIRED and it's associated metadata sidecar `samples.json` file is RECOMMENDED.
```

I'd also include in info about the sidecar json file, is this recommended or required as well?

Reply:
The samples.json file is recommended.
I updated the section to include your feedback. Thanks!